projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -319,14 +320,32 @@ const WeatherHeader = () => {
     if (!isWeatherShown) {
         return <View style={styles.emptyWeatherSpace} />
     }
+
+    return <WeatherController />
+}
+
+const WeatherController = React.memo(() => {
```

Comment:
nit: in general I've seen the term `Container` rather than `Controller`, since we're not in an MVC framework. Ie. `WeatherContainer`.

Reply:
Will change!